editor/src/messages/tool/transform_layer/transform_layer_message_handler.rs

```diff
@@ -982,4 +982,28 @@ mod test_transform_layer {
 		assert!(final_transform.abs_diff_eq(after_scale_transform, 1e-5), "Final transform should match the transform before committing");
 		assert!(!final_transform.abs_diff_eq(original_transform, 1e-5), "Final transform should be different from original transform");
 	}
+	#[tokio::test]
+	async fn test_grs_single_anchor() {
+		let mut editor = EditorTestUtils::create();
+		editor.new_document().await;
+		editor.drag_tool(ToolType::Pen, 100., 100., 100., 100., ModifierKeys::empty()).await;
+		editor.handle_message(ToolMessage::ActivateTool { tool_type: ToolType::Select }).await;
```

Comment:
Whist this does create a path with one point at the moment, this is a bug (I've now created an issue #2503).

If you create a layer, you can then add a path node with a single point with:
```rs
let modification_type = VectorModificationType::InsertPoint {
	id: PointId::generate(),
	position: DVec2::ZERO,
};
responses.add(GraphOperationMessage::Vector { layer, modification_type });
```